pkgs/tools/audio/beets/common.nix

```diff
@@ -105,13 +94,14 @@ python3Packages.buildPythonApplication {
       confuse
       gst-python
       jellyfish
+      lap
       mediafile
       munkres
       musicbrainzngs
       platformdirs
       pyyaml
-      unidecode
       typing-extensions
+      unidecode
```

Comment:
What is the purpose of this change?

Reply:
I ordered the list alphabetically, `unidecode` was the only item in the list not already in alpha order.